shared_conf.py

```diff
@@ -102,11 +110,13 @@
 # -- Options for EPUB output
 epub_show_urls = "footnote"
 
+def add_js_page(app, page_name, _, __, ___):
+    app.add_js_file('carousel.js')
```

Comment:
This is probably something that the directive ought to do, and only for the page it is working on, otherwise every page gets bloated by loading it for no good reason.